vent/controller/control_module.py

```diff
@@ -123,13 +126,15 @@ def __init__(self):
         ###########################  Threading init  #########################
         # Run the start() method as a thread
         self._loop_counter = 0
-        self._running = False
+        self._running = threading.Event()
```

Comment:
What's the benefit of doing it like this?

Reply:
I was concerned there was some delay between setting the bool to stop the thread, but i was probably wrong about that and the lack of copy was the problem. This is just an explicitly threadsafe way of signaling to start/stop. The only other advantage would be if you wanted to make the controller awaitable -- you can have something `.wait()` on the event until it's set true, so eg. if the coordinator wanted to wait for the controller to start you can do it with an Event.